### DISCUSSION TITLE
Improve GitHub help and documentation

### DISCUSSION THREAD

**Discussion by WebCrew** [community/community#63198] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Hallo Community, Hallo GitHub Team,

All in all, your descriptions **are very good** and **relatively understandable** even for people with less specialist knowledge. However, I think more could and should be done **for people with learning disabilities or people who have difficulty reading due to illness such as sub-types of illiteracy**.

It would still be of immense help if you could also integrate small example or tutorial videos in the future and link them right to the appropriate place in the documentation. 

Or you can use some tool such as https://scribehow.com/ to present everything **in a more visual and comprehensible way** and that's even a lot less effort than creating videos.

What do You think about it? Your opinion on the subject, possibly also from those affected, would help a lot. **Thank you** for your interest.

**Best regards**, @WebCrew ;)

**Comment by ghostinhershell** [comment]
Hey there @WebCrew, and thank you so much for reaching out in our Community! 🚀 

Can you give me an example of what you mean here? For example is this feedback related to how our Docs are written? I would love to pass this feedback along to the relevant teams here at GitHub.

Thanks again!

**Reply by WebCrew** [reply]
Yes @ghostinhershell , it's about the Docs. Your documents are good, **even very good** if you have the necessary expertise. But _there are many end users who do not have this expertise_ or _have health or learning issues_, for them it would **be great and super useful** if you could expand your documentation with small tutorial videos or use a tool like the one I linked to you (is not an affiliate link) to present the written docu visually and practically. 

Kind regards and have a nice day ;)

Andreas
